testdata/xml-data/generic_project_richtext.xml

```diff
@@ -0,0 +1,122 @@
+<?xml version='1.0' encoding='utf-8'?>
+
+<knora xmlns="https://dasch.swiss/schema"
+       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:schemaLocation="https://dasch.swiss/schema ../../src/dsp_tools/resources/schema/data.xsd"
+       shortcode="9999"
+       default-ontology="onto">
+
+    <!-- This file contains test cases to ensure that richtext is correctly parsed during an xmlupload -->
+
+    
+    <!-- Resources that are referenced by other resources -->
+
+    <resource
+            label="target_resource_with_iri"
+            restype=":ClassWithEverything"
+            id="target_resource_with_iri"
+            iri="http://rdfh.ch/9999/DiAmYQzQSzC7cdTo6OJMYA"
+    />
+
+    <resource label="target_resource_with_id" restype=":ClassWithEverything" id="target_resource_with_id"/>
+
+    <!-- Resources containing test cases -->
+
+    <resource label="res_with_tags_in_text" restype=":ClassWithEverything" id="res_with_tags_in_text">
+        <text-prop name=":testRichtext">
+            <text encoding="xml">
+                This is
+                <em>italicized and
+                    <strong>bold</strong>
+                </em>
+                text!
+            </text>
+        </text-prop>
+    </resource>
+
+    <resource label="res_text_wrapped_in_tag" restype=":ClassWithEverything" id="res_text_wrapped_in_tag">
+        <text-prop name=":testRichtext">
+            <text encoding="xml">
+                <p>
+                    Paragraph text
+                </p>
+            </text>
+        </text-prop>
+    </resource>
+
+    <resource label="res_with_multiple_paragraphs" restype=":ClassWithEverything" id="res_with_multiple_paragraphs">
+        <text-prop name=":testRichtext">
+            <text encoding="xml">
+                <p>
+                    Paragraph 1 text
+                </p>
+                <p>
+                    Paragraph 2 text
+                </p>
+            </text>
+        </text-prop>
+    </resource>
+
+    <resource label="res_with_escaped_characters" restype=":ClassWithEverything" id="res_with_escaped_characters">
+        <text-prop name=":testRichtext">
+            <text encoding="xml">
+                text &lt;not a tag&gt; text and with ampersand &amp;
```

Comment:
Not sure if syntactically, this would be a candidate for a tag. To make it more tag-like, we could put it like this:

```suggestion
                text &lt;notatag&gt; text and with ampersand &amp;
```